indico/web/client/styles/modules/categories/_category_statistics.scss

```diff
@@ -0,0 +1,28 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.ui.segments > .segment {
+    width: 50%;
+}
+
+.chart-container {
+    width: 480px;
+    height: 300px;
+}
+
+.chart > svg {
+    width: 500px !important;
+    height: 300px !important;
+}
+
+td {
```

Comment:
this is way too generic, and breaks all kinds of things (e.g. many other tables in indico such as the programme code assignment dialog)

Reply:
working on it. :smiley:

Reply:
Opened an Issue on Github: https://github.com/tannerlinsley/react-charts/issues/58

Reply:
Why don't you just use `.ReactChart.chart .tooltip-wrap td`?

Reply:
And I don't think there is another way to customize the color (https://github.com/tannerlinsley/react-charts/blob/a454216a5cdbf30b401200892a52b515a14d516f/src/components/Tooltip.js#L387)

Reply:
` :global(.tooltip-wrap) td` worked :tada: